watertap/examples/flowsheets/case_studies/wastewater_resource_recovery/supercritical_sludge_to_gas/supercritical_sludge_to_gas.py

```diff
@@ -52,16 +52,19 @@ def main():
     initialize_system(m)
 
     results = solve(m)
-    assert_optimal_termination(results)
-    display_results(m)
+    check_optimal_termination(results)
```

Comment:
```suggestion
    if check_optimal_termination(results):
        pass
    else:
        raise AssertionError("The model failed to converge to an optimal solution before adding costing")
```

I just made up a quick example for an error message, but basically, @andrewlee94 originally suggested providing the user with user-friendly error messages (rather than just using `assert_optimal_termination`)

Reply:
With that said, I don't mind keeping the `assert_optimal_termination` for now and then later changing all flowsheets for AMO at the same time in one PR for more uniform messaging. However, if you keep `check_optimal_termination` here, we should include an if-else statement.